docs/release-notes/2.18.0.md

```diff
@@ -0,0 +1,93 @@
+# Percona Monitoring and Management 2.18.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">June 1, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights
+
+The goal for this small release was to eliminate some bugs and complete some features.
+
+- DBaaS
+
+    Added the ability for PMM to install the latest versions of the K8s operator into the K8s cluster. There is no longer any need to install the operator manually. Just connect PMM to your K8s cluster and let PMM do the rest.
+
+- Backup Management
+
+    Backup functionality was released as a Technical Preview feature and will require specific prerequisites from the user side to be installed. Currently, PMM will allow you to:
+
+    - manage storage for backups (S3 only);
+    - execute a backup for MySQL and Mongo instances;
+    - restore a MySQL backup to the same instance from where it was taken (via the UI).
+
+    Restore in other cases is not yet implemented on the UI.
+
+## New Features
+
+* [PMM-7509](https://jira.percona.com/browse/PMM-7509): Integrated Alerting: Pagination for Alerts list
+
+
+
+## Improvements
+
+* [PMM-8029](https://jira.percona.com/browse/PMM-8029): DBaaS: PSMDB 1.8v operator support
+* [PMM-7548](https://jira.percona.com/browse/PMM-7548): Integrated Alerting: Disable edit and delete buttons for templates with source USER_FILE
+
+
+
+## Bugs Fixed
+
+* [PMM-8132](https://jira.percona.com/browse/PMM-8132): DBaaS: Monitoring of PXC cluster stops working after upgrade from 2.16.0 to 2.18.0
+* [PMM-8127](https://jira.percona.com/browse/PMM-8127): DBaaS: DB Clusters disappear when upgrading from 2.16.0 to 2.18.0
+* [PMM-8058](https://jira.percona.com/browse/PMM-8058): IA: Firing alerts disappear after docker restart
+* [PMM-8069](https://jira.percona.com/browse/PMM-8069): IA: Alert template .yaml is not recognized
+* [PMM-8053](https://jira.percona.com/browse/PMM-8053): 'An unexpected error happened' when accessing Settings page as non-admin
+* [PMM-7916](https://jira.percona.com/browse/PMM-7916): DBaaS: Wrong required resources when editing a cluster
+* [PMM-7782](https://jira.percona.com/browse/PMM-7782): Creating new notifications, while older ones are vanishing.
```

Comment:
this is resolved as "Won't Fix" and comments in ticket indicate we did nothign for it...should have been ignored by jirabot...

Reply:
It should not have fixVersion as it's not existed in 2.18 at all. 
removed version from Jira and it's need to be removed from the list too